packages/user-interface/src/components/TableList.module.scss

```diff
@@ -11,6 +11,28 @@
     color: var(--nlportal-heading-5-color, hsl(207 80% 35%));
   }
 
+  &__link-label {
+    border: 0;
+    padding: 0;
+    margin: 0;
+    position: absolute !important;
+    height: 1px;
+    width: 1px;
+    overflow: hidden;
+
+    /* IE6, IE7 - a 0 height clip, off to the bottom right of the visible 1px box */
+    clip: rect(1px 1px 1px 1px);
+
+    /* maybe deprecated but we need to support legacy browsers */
+    clip: rect(1px, 1px, 1px, 1px);
```

Comment:
Nog maar ongeveer 0.6% van de gebruikers wereldwijd gebruikt IE. Het lijkt mij niet wenselijk om deprecated CSS properties te gebruiken om zo weinig gebruikers nog te ondersteunen. We houden in de rest van de code ook geen rekening meer met IE.